lib/search/query_components/query.rb

```diff
@@ -38,17 +55,31 @@ def query_hash
       }
     end
 
-    def base_query
-      return { match_all: {} } if search_term.nil?
-
-      if search_params.enable_new_weighting?
-        core_query = QueryComponents::TextQuery.new(search_params).payload
-      else
-        core_query = QueryComponents::CoreQuery.new(search_params).payload
-      end
-
-      boosted_query = QueryComponents::Booster.new(search_params).wrap(core_query)
-      QueryComponents::Popularity.new(search_params).wrap(boosted_query)
+    def more_like_this_query_hash
```

Comment:
Do you mean the `more_like_this_query_hash`? It's a type of query (like a normal text query) which appears inside the `query` block to be sent to Elasticsearch and is therefore an alternative to `search_query_hash`.